be/src/exec/tablet_sink.h

```diff
@@ -245,6 +245,8 @@ class OlapTableSink : public DataSink {
     int64_t _automatic_bucket_size = 0;
     std::set<int64_t> _immutable_partition_ids;
     RuntimeState* _state = nullptr;
+
+    PLoadChannelProfileConfig _load_channel_profile_config;
```

Comment:
PLoadChannelProfileConfig is a protobuf object? if not so, change another name or just remove prefix "P"?

Reply:
yes, it's a protobuf object